SECURITY.md

```diff
@@ -0,0 +1,22 @@
+# Security Policy
```

Comment:
@joelsmith looks like we already have a SECURITY.md for k/k at https://github.com/kubernetes/kubernetes/blob/master/.github/SECURITY.md

Reply:
Also shows up here - https://github.com/kubernetes/kubernetes/security/policy